src/test.js

```diff
@@ -593,6 +594,14 @@ Test.prototype = {
 			return false;
 		}
 
+		return this.filterMatch();
+	},
+
+	filterMatch: function() {
```

Comment:
I believe this is unrelated clean up, is that right? It's fine, but I just want to check if I may have missed something here that was meant to change behaviour (e.g. the order is not significant).

Reply:
Correct, this was a clean up to split up the responsibility of the function. No behavior change.